deepchem/data/data_loader.py

```diff
@@ -412,5 +412,5 @@ def featurize(self, input_files, in_memory=True):
       return NumpyDataset(images)
     else:
       # from_numpy currently requires labels. Make dummy labels
-      labels = np.zeros(len(images))
+      labels = np.zeros((len(images), 1))
```

Comment:
This is a bugfix to `ImageLoader` to make the dummy labels needed for `DiskDataset.from_numpy` have the right shape.